dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -419,6 +430,19 @@ private LambdaEvent createBasicEvent(String repository, String gitReference, Str
         return updatedServices;
     }
 
+    /**
+     * Decide whether a gitReference is excluded, given a workflow/service's filters
+     * @param gitRefPath Path.of(gitReference) for glob matching with PathMatcher
+     * @param filters Filters specified for a workflow/service in .dockstore.yml
+     * @return
+     */
+    private boolean filterGitReference(final Path gitRefPath, final List<String> filters) {
+        return !filters.isEmpty() && !filters.stream().anyMatch(filter -> {
```

Comment:
I find double negatives harder to understand; if you reverse the return value of the function per my other suggestion, then you wouldn't need the negations here.